docs/refguide/observer-component.md

```diff
@@ -164,7 +164,6 @@ import {observer} from "mobx-react";
 
 * `componentWillReact` doesn't take arguments
 * `componentWillReact` won't fire before the initial render (use `componentWillMount` instead)
-* `componentWillReact` won't fire when receiving new props or after `setState` calls (use `componentWillUpdate` instead)
```

Comment:
I am thinking it would be better to just change  `won't fire` into `will fire`. It's useful to leave this note here, because a lot of programmers won't know how react-mobx works under the hood and they would expect this to be true.